bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/ttH_HToTT/templatecards_run3/ttH_proc_card.dat

```diff
@@ -0,0 +1,11 @@
+# import model sm-no_b_mass
+import model sm
```

Comment:
why not using directly the no_b_mass model, if we also discussed this in UL PR please add the PR link here

Reply:
Hi @menglu21, it's like this because this is what was used in the Run 2 MC, but we don't think it shouldn't make a difference so we are open to suggestion if you think it's better to change it.

Reply:
please change to the no_b mass model, thanks. same for the Run2 one

Reply:
Sure! Switched to sm-no_b_mass model for both run2UL and run3 now

Reply:
> please change to the no_b mass model, thanks. same for the Run2 one

Hi @menglu21 , the change has been made as requested